src/playground/with-layout/autocomplete/autocomplete-form.component.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright Akveo. All Rights Reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { Observable, of } from 'rxjs';
+
+import { FormControl } from '@angular/forms';
+import { map, startWith } from 'rxjs/operators';
+
+@Component({
+  selector: 'nb-autocomplete-showcase',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  templateUrl: './autocomplete-form.component.html',
+})
+export class AutocompleteFormComponent implements OnInit {
+
+  options: string[];
+  filteredOptions$: Observable<string[]>;
+  inputFormControl: FormControl;
+
+  ngOnInit() {
+
+    this.options = ['Option 1', 'Option 2', 'Option 3'];
+    this.filteredOptions$ = of(this.options);
+
+    this.inputFormControl = new FormControl();
+
+    this.filteredOptions$ = this.inputFormControl.valueChanges
+      .pipe(
+        startWith(''),
+        map(item => this.filter(item)),
```

Comment:
Let's rename `item` to `filterString` or something more descriptive. Otherwise it's hard to understand. Also update this in all examples.

Reply:
I agree, renamed